setupAdapters.js

```diff
@@ -0,0 +1,25 @@
+/* eslint global-require: 0 */
+/**
+ * This file is needed only because we run our unit tests on multiple
+ * versions of React at a time. This file basically figures out which
+ * version of React is loaded, and configures enzyme to use the right
+ * corresponding adapter.
+ */
+const Version = require('./src/version');
+const Enzyme = require('./src');
+
+let Adapter = null;
+
+if (Version.REACT013) {
+  Adapter = require('./src/adapters/ReactThirteenAdapter');
```

Comment:
It's not part of this PR, but it's absolutely the plan. See "Followup PR moving the adapters into their own packages" in the OP.